src/api/processhub-api.ts

```diff
@@ -0,0 +1,329 @@
+/* eslint-disable @typescript-eslint/indent */
```

Comment:
it would be better to disable the rules directly at the lines where necessary.
Why is the indentation rule disabled?